source/extensions/filters/http/aws_lambda/aws_lambda_filter.cc

```diff
@@ -260,6 +237,7 @@ Http::FilterDataStatus Filter::decodeData(Buffer::Instance& data, bool end_strea
   setLambdaHeaders(*request_headers_, arn_->functionName(), invocation_mode_);
   const auto hash = Hex::encode(hashing_util.getSha256Digest(decoding_buffer));
   sigv4_signer_->sign(*request_headers_, hash);
+  stats().upstream_rq_payload_size_.recordValue(decoding_buffer.length());
```

Comment:
I don't see this tested anywhere. I would at least have a mock expect that this is being called.